wormhole-connect/src/config/index.ts

```diff
@@ -59,34 +64,32 @@ export const WORMHOLE_API =
 
 export const EXPLORER = config.explorer;
 
-export const ATTEST_URL =
-  ENV === 'MAINNET'
-    ? 'https://www.portalbridge.com/#/register'
-    : ENV === 'DEVNET'
-    ? ''
-    : 'https://wormhole-foundation.github.io/example-token-bridge-ui/#/register';
+export const ATTEST_URL = {
+  MAINNET: 'https://www.portalbridge.com/#/register',
```

Comment:
should be https://portalbridge.com/advanced-tools/#/register

probably would be good to handle it in a separated ticket